src/envo/envo-edit.owl

```diff
@@ -40171,6 +39843,535 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3400002> "ar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3400002> <http://purl.obolibrary.org/obo/ENVO_00000112>)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3400002> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000086> <http://purl.obolibrary.org/obo/ENVO_01000238>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510000> (wood dust)
```

Comment:
The definition and placement look good - but wouldn't there be subclass axioms following the textual definition? Using `RO:composed primarily of` and some CHEBI [polysaccharides](http://purl.obolibrary.org/obo/CHEBI_18154)?

Reply:
Pier, I would trust you on this; if you want 

```
SubclassOf(ENVO:3510000 RO:composed primarily of some CHEBI polysaccharides)
```
To be added to the def I shall. For our use case, the existence of the term is more important, not the axiomatisation. But we are happy to follow your lead on this!

Reply:
I dont feel confident to add this axiom, so I will leave it for this PR unless told otherwise.